doc/configure

```diff
@@ -111,30 +109,36 @@ def roff_to_html(man, roff_out, builds):
     return html_man
 
 
-def convert_man_dir_to_roff(src_dir, dst_dir, rule, html_mans, builds):
+def convert_man_dir_to_man(src_dir, dst_dir, rule, html_mans, builds):
     for man in (src_dir).iterdir():
-        roff_out = dst_dir / f"{man.stem}.roff"
+        with open(man) as fp:
+            line = fp.readline().rstrip()
+        index = line[line.find('(')+1:line.find(')')]
```

Comment:
I don't think this works for the YAML files under voluptuous-man. Those files don't have a first line with a chapter number.

I think the three lines above should be moved to inside the `if` statement below. They are only applicable to `.scdoc` files.